torch2jax/__init__.py

```diff
@@ -916,6 +916,24 @@ def relu(x, inplace=False):
     return Torchish(jax.nn.relu(_v(x)))
 
 
+@implements(torch.nn.functional.prelu)
+def prelu(a: Torchish, weight: Torchish):
```

Comment:
pytorch calls the first argument "input": https://docs.pytorch.org/docs/stable/generated/torch.nn.functional.prelu.html

Would this work with eg `t2j(lambda x: torch.nn.functional.prelu(input=x, weight=123))`?